main.js

```diff
@@ -0,0 +1,165 @@
+// 1
+
+function getDeltaFunction(numbersObject) {
+    let a = numbersObject.a;
+    let b = numbersObject.b;
+    let c = numbersObject.c;
+
+    function getDelta(newNumbersObject = {}) {
+        if (newNumbersObject.a !== undefined) {
```

Comment:
Use the nullish coalescing operator instead 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing